pkg/api/types.go

```diff
@@ -121,3 +133,36 @@ type InstallResult struct {
 	// Error describes last error encountered during install operation
 	Error error
 }
+
+// SourceInfo stores information about the source code
+type SourceInfo struct {
+	// Ref represents a commit SHA-1, valid GIT branch name or a GIT tag
+	// The output image will contain this information as 'io.openshift.build.commit.ref' label.
```

Comment:
Actually now it's `io.openshift.s2i.build.commit.ref`, rewrite it to say `DefaultNamespace + 'commit.ref'`.
This also applies to all other `SourceInfo` fields.